tox.ini

```
@@ -1,33 +1,33 @@
 [tox]
-minversion = 2.4
 envlist =
-    py36
-    py37
-    py38
-    pypy3
-    coverage-report
+    flake8,
+    py36,
+    py37,
+    py38,
+    pypy3,
+    coverage,
+minversion = 2.4
 
 [testenv]
 usedevelop = True
-commands =
-    py.test --cov=src --cov-report=xml {posargs}
-setenv =
-  COVERAGE_FILE=.coverage.{envname}
 extras = test
 deps =
-    pytest >= 5.4.2
-    pytest-cov
-    pytest-flake8
+    pytest
     pytest-remove-stale-bytecode
+commands = py.test []
 
-[testenv:coverage-report]
-deps = coverage
-setenv =
-  COVERAGE_FILE=.coverage
-skip_install = true
+[testenv:coverage]
+basepython = python3
+deps =
+    {[testenv]deps}
+    pytest-cov
+    coverage
+    coverage-python-version
 commands =
-    coverage erase
-    coverage combine
-    coverage report
-    coverage html
-    coverage xml
+    py.test --cov=src --cov-report=html []
+
+[testenv:flake8]
+basepython = python3.8
```

```suggestion
basepython = python3
```

This change allows to switch to another Python version in GHA later on without having to change here, too + it is in sync with `[testenv:coverage]` which also uses `python3`.